src/lib.rs

```diff
@@ -98,6 +99,26 @@ impl PGAppend<*mut i8> for pg::StringInfo {
     }
 }
 
+struct Wrapped(pg::Enum_ReorderBufferChangeType);
+
+impl fmt::Display for Wrapped {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        use pg::Enum_ReorderBufferChangeType::*;
+        let formatted_token = match self.0 {
+            REORDER_BUFFER_CHANGE_INSERT => "insert",
+            REORDER_BUFFER_CHANGE_UPDATE => "update",
+            REORDER_BUFFER_CHANGE_DELETE => "delete",
+            REORDER_BUFFER_CHANGE_INTERNAL_SNAPSHOT => "internal_snapshot",
+            REORDER_BUFFER_CHANGE_INTERNAL_COMMAND_ID => "internal_command_id",
+            REORDER_BUFFER_CHANGE_INTERNAL_TUPLECID => "internal_tuplecid",
+            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_INSERT => "internal_spec_insert",
+            REORDER_BUFFER_CHANGE_INTERNAL_SPEC_CONFIRM => "internal_spec_confirm",
+            REORDER_BUFFER_CHANGE_MESSAGE => "change_message"
```

Comment:
So this is kind of weird, and I feel I should explain it.

This is a value in the postgres enum, but only for version 9.4. So on 9.4 this correctly generates all paths (and allows the code to be built).

On 9.5+ this acts as a "catch all" (this technically goes unused since we have all the previous values defined), but luckily rust still accepts it as a named `_` and allows the code to build.

Really once 9.4 support is no longer needed you can drop this.